hooks/useProfile.ts

```
@@ -20,17 +20,20 @@ const saveIndentify = async (
 };
 
 const useProfile = () => {
+  const usernameRef = useRef() as any;
+  const bioRef = useRef() as any;
+  const statusRef = useRef() as any;
+  const { getProfile, connectedAddress } = useUserProvider();
   const { data, isLoading } = useArtistProfile();
-  const { connectedWallet } = useConnectedWallet();
   const [socialAccounts, setSocialAccounts] = useState<any>(null);
```

ESLint

[error] 'setSocialAccounts' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)